python/lsst/ts/imsim/utils/plotUtil.py

```diff
@@ -51,7 +47,7 @@ def _save_fig(
 
 
 def plot_fwhm_of_iters(
-    pssn_files: List[str], save_to_file_path: Optional[str] = None, dpi: None = None
+    pssn_files: list[str], save_to_file_path: str = None, dpi: None = None
```

Comment:
See comment above

Reply:
Okay, found a few other instances of this and fixed them up.